services/cd-service/pkg/repository/repository.go

```diff
@@ -434,11 +434,11 @@ func (r *repository) ProcessQueueOnce(ctx context.Context, e element, callback P
 			err = fmt.Errorf("failed to push - this indicates that branch protection is enabled in '%s' on branch '%s'", r.config.URL, r.config.Branch)
 		}
 	}
-	span, _ := tracer.StartSpanFromContext(ctx, "PostPush")
+	span, _ := tracer.StartSpanFromContext(e.ctx, "PostPush")
```

Comment:
You also have to pass the resulting context to `sendWebhookToArgoCd`, don't you?

Reply:
done